tests/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const CACHED_COMMUNITY_GUILD_ID = 907350958810480671n;
+export const CACHED_COMMUNITY_GUILD_ID = 907350958810480671n; // revert back to 907350958810480671n when testing is finished
```

Comment:
remove this constant. Also maybe open an issue to move this to an env.

Reply:
that const was already here and i bet its used in other places too

Reply:
Still remove the comment, I might change to env then